src/renderers/imgui_dx11/device_and_swapchain.rs

```diff
@@ -79,28 +77,28 @@ impl DeviceAndSwapChain {
     ) -> Self {
         let back_buffer = unsafe {
             let p_back_buffer: ID3D11Resource = swap_chain.GetBuffer(0).expect("GetBuffer");
+            let mut back_buffer = None;
 
-            let back_buffer = dev
-                .CreateRenderTargetView(&p_back_buffer, null_mut())
+            dev.CreateRenderTargetView(&p_back_buffer, None, Some(&mut back_buffer))
                 .expect("CreateRenderTargetView");
 
-            dev_ctx.OMSetRenderTargets(&[Some(back_buffer.clone())], None);
+            dev_ctx.OMSetRenderTargets(Some(&[Some(back_buffer.clone().unwrap())]), None);
```

Comment:
If Hawkwood was a Rust developer, I'm sure having to do this would give him conniptions.